### ISSUE TITLE
There is no loading percentage when exporting frames

### ISSUE THREAD

**Issue by MirzaIkhsan** [Baseflow/screenrecorder#23]
I think it would be nice if there is some percentage when exporting frames so I can show the progress to the user. Or maybe there are some alternative that I can use to show the percentage to user?

Thanks 😄

**Comment by ueman** [comment]
Yeah, this would certainly be nice. I'll gladly accept such a thing as PR. 

I'm not entirely sure, but I believe on web this wouldn't help since the processing is currently a blocking operation.
